ui-account/build.gradle

```diff
@@ -83,7 +83,5 @@ dependencies {
 
     implementation Libs.Hilt.library
     kapt Libs.Hilt.compiler
-
-    implementation Libs.AndroidX.Hilt.viewmodel
     kapt Libs.AndroidX.Hilt.compiler
```

Comment:
You don't seem to have `implementation Libs.AndroidX.Hilt.work` in this module. You can also remove `kapt Libs.AndroidX.Hilt.compiler`